### PR TITLE
Fix stake amount

### PR DESCRIPTION
**Pull request by vaibhavchellani**

### CODE REVIEW

File: bazooka/utils.go [hunk 1]
```diff
@@ -150,3 +150,11 @@ func (b *Bazooka) GetReceipt(txHash common.Hash) (*types.Receipt, error) {
...
 	return client.TransactionReceipt(context.Background(), txHash)
 }
+
+// func etherToWei(val *big.Int) *big.Int {
+// 	return new(big.Int).Mul(val, big.NewInt(params.Ether))
+// }
+
+// func weiToEther(val *big.Int) *big.Int {
+// 	return new(big.Int).Div(val, big.NewInt(params.Ether))
+// }
```
Comment (ChihChengLiang) [inline bazooka/utils.go:160]: nitpick: Can remove these

File: cmd/tx.go [hunk 2]
```diff
@@ -521,9 +520,12 @@ func validateAndTransfer(DBI *db.DB, bazooka *bazooka.Bazooka, fromIndex, toInde
 	if bal.Int64() <= int64(amount+fee) {
 		return "", ErrInvalidAmount
 	}
+	fmt.Println("here")
```
Comment (ChihChengLiang) [inline cmd/tx.go:523]: nitpick: Unhelpful print

File: cmd/tx.go [hunk 3]
```diff
@@ -521,9 +520,12 @@ func validateAndTransfer(DBI *db.DB, bazooka *bazooka.Bazooka, fromIndex, toInde
...
+	fmt.Println("fee", int64(fromIndex), int64(toIndex), int64(fee), nonce.Int64(), int64(amount), core.TX_TRANSFER_TYPE)
```
Comment (ChihChengLiang) [inline cmd/tx.go:524]: nitpick: should use log

File: cmd/tx.go [hunk 4]
```diff
@@ -521,9 +520,12 @@ func validateAndTransfer(DBI *db.DB, bazooka *bazooka.Bazooka, fromIndex, toInde
...
 
 	txData, err := bazooka.EncodeTransferTx(int64(fromIndex), int64(toIndex), int64(fee), nonce.Int64(), int64(amount), core.TX_TRANSFER_TYPE)
 	if err != nil {
+		fmt.Println("here", err)
```
Comment (ChihChengLiang) [inline cmd/tx.go:528]: nitpick: Unhelpful print

**Review by ChihChengLiang — APPROVED** [review]
LGTM. Feel free to address the nitpicks now or later.

**Comment by vaibhavchellani** [comment]
All nitpicks will be resolved in the next PR.

**Pull request merged**